Bottom: 41f91d96e5d57ac7f858ecf93a1a835c32a3cabb
Top:    bde9309ec15274e805604b00dcea440422688490
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-07 00:06:08 +0100

TEMP TEST


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index ea77412..878cbf3 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -280,6 +280,7 @@ have_crosshair_attachments (void)
 static gboolean
 ghid_idle_cb (gpointer data)
 {
+  return;
   if (Settings.Mode == NO_MODE)
     SetMode (ARROW_MODE);
   ghid_mode_cursor (Settings.Mode);
